.gitignore

```diff
@@ -9,4 +9,8 @@ packages/*/coverage
 test.sqlite
 .DS_Store
 *.log
-tsconfig.build.tsbuildinfo
\ No newline at end of file
+tsconfig.build.tsbuildinfo
+.dev.env
+#*#
```

Comment:
wasn't sure what this line was for so i did a quick google search & it turned up that it might need to be escaped?
https://stackoverflow.com/questions/2209810/add-glob-to-gitignore

Reply:
yes! thanks. my template file is ancient, thought maybe this never worked correctly.